youtube.js

```diff
@@ -179,10 +179,13 @@ $("body").unbind("click").on("click", "yt-live-chat-text-message-renderer,yt-liv
 });
 
 function hideActiveChat() {
+  var html = '<div class="hl-c-cont fadeout"></div>';
   if(sessionID) {
     var remote = {
       version: version,
-      command: "hide",
+      command: "show",
```

Comment:
I'd rather keep this commend set to "hide" for this, but including the config object here is a good idea. Can you change this back to "hide" and remove the "html" property? I'll update the remote window to make sure it accepts config settings even for the "hide" message if it doesn't already.

Reply:
It does not currently work with "hide" .... I think the pushstream.js code on the receiving end ignores the config information fir a "hide" message. I f you change pushstream.js, then "hide" would be fine here.

Reply:
Thanks for confirming. I just modified the remote window JS to load the config in a "hide" message as well.